requirements-dev.txt

```diff
@@ -5,4 +5,5 @@ autopep8==1.3.1
 nose==1.3.7
 coverage==4.4.2
 mock==2.0.0
-pylint==1.8.1
\ No newline at end of file
+pylint==1.8.1
+six>=1.10.0
```

Comment:
Please lock this to the version (use "==" instead of ">=")

Reply:
Would you accept: `six<2,>=1.10.0`?  Which would lock it to a *Major* version of `1.`

I chose that minimum version because of upstream dependencies.  As you stated python is not your primary language, Six is highly unlikely to have major changes, and hard locked dependencies will likely cause dependency conflicts with other packages, because pinned versions will actually uninstall other versions.

It does look like we may get a dependency resolver this year, but I would highly recommend pulling the min version of shared dependencies from your upstream requirements and only pinning on failure.  https://github.com/pypa/pip/issues/988

let me know if you do want these hard pinned, and I will do so.